include/sd_sector_addresses.h

```diff
@@ -22,122 +22,251 @@ struct SectorAddress
+#define SIT_4_WALKMENU_ACTIVATED SectorAddress { 0x0000, 0x1951 }
 
-#define SINGLE_STEP_LARGE SectorAddress { 0x0000, 0x0654 }
-/***** Walk menu *****/
+#define HOME_WAlKMENU SectorAddress { 0x0000, 0x31AE }
 
 /**********
- * Sit menu
+ * Single step menu + home stand + sit + home
  **********/
-#define SIT SectorAddress { 0x0000, 0x222C }
-#define SIT_SELECTED SectorAddress { 0x0000, 0x0B13 }
-#define SIT_ACTIVATED SectorAddress { 0x0000, 0x0AC2 }
+#define SINGLE_STEP_4 SectorAddress { 0x0000, 0x30BB }
+#define SINGLE_STEP_4_SELECTED SectorAddress { 0x0000, 0x1627 }
+#define SINGLE_STEP_4_ACTIVATED SectorAddress { 0x0000, 0x1678 }
 
-#define HOME_SIT SectorAddress { 0x0000, 0x218A }
-#define HOME_SIT_SELECTED SectorAddress { 0x0000, 0x00A2 }
-#define HOME_SIT_ACTIVATED SectorAddress { 0x0000, 0x0051 }
+#define HOMESTAND_4_SSMENU SectorAddress { 0x0000, 0x310C }
+#define HOMESTAND_4_SSMENU_SELECTED SectorAddress { 0x0000, 0x180D }
+#define HOMESTAND_4_SSMENU_ACTIVATED SectorAddress { 0x0000, 0x185E }
 
-#define HOME_STAND SectorAddress { 0x0000, 0x21DB }
-#define HOME_STAND_SELECTED SectorAddress { 0x0000, 0x0288 }
-#define HOME_STAND_ACTIVATED SectorAddress { 0x0000, 0x0237 }
+#define SIT_4_SSMENU SectorAddress { 0x0000, 0x315D }
+#define SIT_4_SSMENU_SELECTED SectorAddress { 0x0000, 0x1900 }
+#define SIT_4_SSMENU_ACTIVATED SectorAddress { 0x0000, 0x1951 }
 
-#define TURN_OFF SectorAddress { 0x0000, 0x227D }
-#define TURN_OFF_SELECTED SectorAddress { 0x0000, 0x1491 }
-#define TURN_OFF_ACTIVATED SectorAddress { 0x0000, 0x1440 }
-/***** Sit menu *****/
+#define HOME_WAlKMENU SectorAddress { 0x0000, 0x31AE }
 
 /**********
- * Obstacle menu
+ * Obstacle menu - normal
  **********/
-#define SOFA SectorAddress { 0x0000, 0x2505 }
+#define SOFA SectorAddress { 0x0000, 0x0001 }
+
+#define SOFA_SIT SectorAddress { 0x0000, 0x0052 }
+#define SOFA_SIT_SELECTED SectorAddress { 0x0000, 0x00A3 }
+#define SOFA_SIT_ACTIVATED SectorAddress { 0x0000, 0x00F4 }
+
+#define SOFA_STANDUP SectorAddress { 0x0000, 0x0145 }
+#define SOFA_STANDUP_SELECTED SectorAddress { 0x0000, 0x0196 }
+#define SOFA_STANDUP_ACTIVATED SectorAddress { 0x0000, 0x01EF }
+
+
+#define SLALOM SectorAddress { 0x0000, 0x3529 }
+
+#define WALK_2O SectorAddress { 0x0000, 0x357A }
+#define WALK_2O_SELECTED SectorAddress { 0x0000, 0x02DA }
+#define WALK_2O_ACTIVATED SectorAddress { 0x0000, 0x032B }
+
+
+#define ROUGH_TERRAIN SectorAddress { 0x0000, 0x037C }
+
+#define HIGHSTEP_RT SectorAddress { 0x0000, 0x03CD }
+#define HIGHSTEP_RT_second SectorAddress { 0x0000, 0x06A6 }
+#define HIGHSTEP_RT_SELECTED SectorAddress { 0x0000, 0x041E }
+#define HIGHSTEP_RT_ACTIVATED SectorAddress { 0x0000, 0x046F }
+
+#define SMALLSTEP_RT SectorAddress { 0x0000, 0x04C0 }
+#define SMALLSTEP_RT_second SectorAddress { 0x0000, 0x0799 }
+#define SMALLSTEP_RT_SELECTED SectorAddress { 0x0000, 0x0511 }
+#define SMALLSTEP_RT_ACTIVATED SectorAddress { 0x0000, 0x0562 }
+
+//#define MIDDLESTEP_RT SectorAddress { 0x0000, 0x05B3 }
```

Comment:
I think this can be removed?

Reply:
why?

Reply:
Because it is a comment and is not used anywhere. Unless there is a good reason for this?

Reply:
They are used in the StateMachine:constructRough_TerrainMenu?

Reply:
I am talking about line 159, the one preceded with `//`. All commented lines are discarded upon compilation, therefore, it is safe to remove this line. And `MIDDLESTEP_RT` is not used in the state machine.

Reply:
You are right, that one was used before movement decided to have multiple middle steps and therefore have other pictures (first, second). So that one can be removed